lib/streamlit/hello/util.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018-2019 Streamlit Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+MAX_DOCSTRING = 1000
+
+
+def remove_declaration_and_docstring(lines):
```

Comment:
You can get a function's docstring with `the_func.__doc__`. Wouldn't it be easier / safer to just use that here?

For example:
```
doc = the_func.__doc__.splitlines()
last_line = len(doc)
return lines[last_line + 1:]
```

Reply:
I can try this. Seems a good option. Not sure on top of my head if `the_func.__doc__.splitlines()` has to maintain the same formatting and gives the same number of lines, but it may as well be.

Reply:
It does not tell me about the lines with `"""`, it only gives me the lines with the actual content of the docstring. So,
```
"""
"""
```
and 
```
"""  """
```
are the same. I would still have to parse and do something with these, so not sure of the actual advantage.

Reply:
I tried to write it and it quickly became as complex. I would leave it as it is. I am not a big fan of ad hoc logic like this one...

Reply:
Honestly there might not be need for doing all this introspection... we could just remove the docstring and factor out the comments, then require all demo functions *not* to have a docstring (easier assert). The current code happened because of some initial back and forth between me and Adrian, but no reason to stay this way. If you like this approach better I can refactor in this sense, it will take an extra hour or so.